CHANGELOG.md

```diff
@@ -4,123 +4,212 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+
+### Added
+
+* Allow SQLite query parameters and support cached databases ([#561][#561])
+* Support for unix socket for aiomysql and asyncmy ([#551][#551])
+
+[#551]: https://github.com/encode/databases/pull/551
+[#561]: https://github.com/encode/databases/pull/546
+
+### Changed
+
+* Bump requests from 2.28.1 to 2.31.0 ([#562][#562])
+* Bump starlette from 0.20.4 to 0.27.0 ([#560][#560])
+* Bump up asyncmy version to fix `No module named 'asyncmy.connection'` ([#553][#553])
+* Bump wheel from 0.37.1 to 0.38.1 ([#524][#524])
+
+[#562]: https://github.com/encode/databases/pull/562
+[#560]: https://github.com/encode/databases/pull/560
+[#553]: https://github.com/encode/databases/pull/553
+[#524]: https://github.com/encode/databases/pull/524
+
+### Fixed
+
+* fix: incorrect concurrent usage of connection and transaction ([#546][#546])
```

Comment:
Could you write a longer description at the top of the release section of the practical effects this would have for existing users?

Reply:
I looked around to see if I could find examples of projects using the keepachangelog format that also had little release notes, but couldn't find any - it also doesn't seem like keepachangelog forbids it though? So I'm not entirely sure how much detail to put in here, but here's a start:

> This release should fix connection and transaction issues with concurrent task execution - that is, using `asyncio.create_task`, not just coroutines. Users should not need to change their code unless they are spawning tasks within an open transaction and wish to operate on it. In this case, the active connection (`db.connection()`) should be explicitly passed as an argument to the created tasks.  Database connections should truly now be task-local and are not inherited by child-tasks. The `@db.transaction` decorator now properly utilizes the current task's connection, rather than that of the task where the decorated function was declared. 

Maybe something like this is better to put in the github release than in the changelog?

Reply:
That works for me. I'll put it in the GitHub release.